src/common/models/Projection.ts

```diff
@@ -342,22 +348,29 @@ export class Projection {
     summaryWithTimeseries: RegionSummaryWithTimeseries,
     futureDaysToInclude: number,
   ) {
-    const earliestDate = moment.min(
-      summaryWithTimeseries.timeseries.map(row => moment.utc(row.date)),
-    );
-    const latestDate = moment.max(
-      summaryWithTimeseries.timeseries.map(row => moment.utc(row.date)),
+    const timeseriesRaw = summaryWithTimeseries.timeseries;
+    const actualsTimeseriesRaw = summaryWithTimeseries.actualsTimeseries;
+    assert(
+      actualsTimeseriesRaw.length > 0,
+      `FIPS ${this.fips} missing actuals timeseries!`,
```

Comment:
My first impression is that I'm not sure i would expect this assertion to be true - are there not counties that don't have any data? (i.e. if they don't have any cases ever?)

Reply:
I added this assert partially to test that hypothesis.  And since `yarn update-location-summaries` doesn't hit the assert, I think it means it is safe for now.  And since the code (currently) relies on the assert passing, we'll need to fix the code if it ever fails, so I think it's good to assert it to catch issues in the future.

To your specific question, my expectation is that either a county will have never had cases, so we should have an actuals timeseries with all `0`s or we will not know if it had cases, in which case we should continue to just not include it in our API (if we don't have actuals or projections, what's the point)?  This seems to be the current state of affairs.  But we can always revisit in the future.